src/compat/db_admin.c

```diff
@@ -951,7 +951,21 @@ db_restart_ex (const char *program, const char *db_name, const char *db_user, co
       db_set_preferred_hosts (preferred_hosts);
     }
 
+#if !defined(CS_MODE)
   return db_restart (program, false, db_name);
+#else
+  tde_Cipher.is_loaded = false;
+  if ((retval = db_restart (program, false, db_name)) == NO_ERROR)
+    {
+      // Even if tde_get_data_keys() fails, it is executed normally.
```

Comment:
What does this comment mean?

Reply:
Even if the TDE value is not obtained normally, it means that the error handling is not done to ensure that the basic function can be performed.

See below page
"https://www.cubrid.org/manual/en/11.0/security.html#tde"


`When TDE is unavailable
In the following cases, the TDE feature cannot be used, and an error occurs because the TDE module cannot be loaded correctly.

When the valid key file cannot be found

When the key set on the database cannot be found in the key file

Even if the TDE module is not loaded, the server can start normally, and users can access unencrypted tables. This means that all DML and DDL such as SELECT and INSERT only for TDE-encrypted tables cannot be executed.

However, the case log data has been encrypted is different. If the log data is encrypted when the TDE module is not loaded and the log is accessed by recovery, HA, VACUUM, etc., the system cannot be properly executed, and the entire server has no option but to stop running the server.`

Reply:
Should the case when the TDE module is not loaded also be handled in this issue?

Reply:
@shparkcubrid
If the TDE module is not loaded, DBLINK's encryption processing cannot work. However, it is intended to enable behavior other than DBLINK.